### PR TITLE
make Simple launch file

### PR DESCRIPTION
**Pull request by Ryujiyasu**
1. separate param file from launch folder
2. delete ros launch files
3. make simple launch file

### CODE REVIEW

File: mavros/launch/node.launch.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from launch.substitutions import ThisLaunchFileDir
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+	config_dir = LaunchConfiguration('config_dir',
+		default=os.path.join(
+			get_package_share_directory('mavros'),
+			'param',
+			"ros2_mavros_params.yaml"
```
Comment (vooon) [inline mavros/launch/node.launch.py:21]: File name doesn't match.

File: mavros/param/mavros_params.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+mavros_node:
+  ros__parameters:
+    fcu_url: "udp://:14540@"
+    gcs_url: ""
+    tgt_system: 1
+    tgt_component: 1
```
Comment (vooon) [inline mavros/param/mavros_params.yaml:6]: Need to port pluginlists and one of configs, e.g. px4.

File: mavros/launch/node.launch.py (hunk 1 above)
Comment (Ryujiyasu) [reply mavros/launch/node.launch.py:21]: changed name "apm_config.yaml"

**Comment by Ryujiyasu** [comment]
@vooon 
which name is correct
px4flow->"px" or "px4flow"

**Comment by vooon** [comment]
https://github.com/mavlink/mavros/blob/341674a66f7dda509bda0ca71b172b9a4efdc67b/mavros_extras/src/plugins/px4flow.cpp#L46

File: mavros/param/config.yaml [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+# log_transfer None
+# mag_calibration_status None
+# manual_control None
+# mocap_pose_estimate None
+# mount_control None
+# nav_controller_output None
+# obstacle_distance None
+
+# odometry:
```
Comment (vooon) [inline mavros/param/config.yaml:86]: Parameter name would be: `<UAS-name>/<plugn-name>`. E.g. `mavros/odometry`.

**Review by vooon — COMMENTED** [review]
Please rebase PR.

File: mavros/param/config.yaml (hunk 3 above)
Comment (Ryujiyasu) [reply mavros/param/config.yaml:86]: change <plugn-name> --> mavros/<plugn-name>

**Review by vooon — CHANGES_REQUESTED** [review]

File: mavros/mavros_plugins.xml [hunk 4]
```diff
@@ -4,138 +4,138 @@ import mavros_cog
 
 mavros_cog.outl_plugins_xml('src/plugins', 'mavros_plugins')
 ]]] -->
-<library path="mavros_plugins">
+<library path="mavros/mavros_plugins">
```
[the remark this answers is not in the document]
Comment (vooon) [reply mavros/mavros_plugins.xml:7]: That file should not be changed at all.

[the remark this answers is not in the document]
Comment (eMrazSVK) [reply mavros/mavros_plugins.xml:7]: Confirmed, I successfully launched `ros2 launch mavros node.launch.py` with unchanged [mavros_plugins.xml](https://github.com/mavlink/mavros/blob/2.2.0/mavros/mavros_plugins.xml) so the changes for this file can be removed/not applied. 

Tested with `ArduCopter V4.2.3-rc2`

@Ryujiyasu

**Comment by vooon** [comment]
Replaced by #1834

**Pull request closed without merging**